charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -30,4 +30,77 @@ spec:
   ingress:
     - {}
   {{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-prometheus
+  namespace: kyverno
+  labels:
+    app: kyverno
+spec:
+  podSelector:
+    matchLabels:
+      app: kyverno
+  {{- if .Values.networkPolicy.prometheusMatchLabels }}
+  ingress:
+  - from:
+    - podSelector:
```

Comment:
To elaborate on my comment, if you do `podSelector` without `namespaceSelector`, that element assumes it's in same namespace as Kyverno.

Reply:
Oh ok, I got it, I think I misunderstood the previous comment...

Reply:
So I should just add a default configuration in the helm chart like this?
```yaml
    - namespaceSelector:
        matchLabels:
          app.kubernetes.io/name: prometheus
```

Reply:
Ya but it would be good to have the labels be a value like `networkPolicy.prometheusNamespaceLabels` so people can easily change it if the defaults don't match their site.